### PR TITLE
added first steps in the workflow

### PR DESCRIPTION
**Pull request by kalimu**

### CODE REVIEW

**Comment by kalimu** [comment]
@maciekbanas Maciej please take a look at the code, and if ok you can move to the #2

File: R/set_llm.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    )
+  }
+
+  invisible(gitai)
+}
+
+#' export
+set_system_prompt <- function(gitai, system_prompt) {
+
+  gitai$llm$system_prompt <- system_prompt
```
Comment (maciekbanas) [inline R/set_llm.R:20]: There is already method for that in `GitAI` - could be something like:

```suggestion
  gitai$set_prompt(system_prompt)
```

**Review by maciekbanas — COMMENTED** [review]
@kalimu Finished review with one question/suggestion.

File: R/set_llm.R (hunk 1 above)
Comment (maciekbanas) [reply R/set_llm.R:20]: What is more, for the time being if we run `set_system_prompt()` just after `initialize_project()` without `set_llm()` (as I did by mistake when testing manually) there is no error returned.

**Review by maciekbanas — COMMENTED** [review]
One more comment on `set_system_prompt()`. Should probably return error if no `llm` is set.

File: R/set_llm.R (hunk 1 above)
Comment (kalimu) [reply R/set_llm.R]: Ok. I've changed how setting prompts works. 
Now we would use set_prompt() function that sets system (but can set also part of user prompt in the future if needed). And it uses system_prompt which is now active binding (for setting and getting the prompt). 
And it rises error if there is no llm set.

**Pull request merged**